next/next.config.js

```diff
@@ -1,8 +1,14 @@
-
 // for transpiling all ESM @fullcalendar/* packages
 // also, for piping fullcalendar thru babel (to learn why, see babel.config.js)
+
 const withTM = require('next-transpile-modules')([
-  '@fullcalendar'
+  // Need to specify all @fullcalendar modules separately
+  // with next-transpile-modules^6.x …
```

Comment:
As mentioned above this PR is good but I'd refer folks to the `next-transpile-modules` [compatibility table](https://github.com/martpie/next-transpile-modules#compatibility-table) as well.